lib/slack/analysis/network.py

```diff
@@ -784,4 +784,4 @@ def identify_hubs_and_experts(log_dict, nicks, nick_same_list):
         if node_name in top_hub:
             message_graph.node[node_name]['shape'] = 'square'
 
-    return message_graph
+    return message_graph, top_hub, top_keyword_overlap, top_auth
```

Comment:
@kaushik-rohit you are changing the function called by driver files. All the drivers would need updating as well.

Reply:
@kaushik-rohit The driver files at the top level need updating for any breaking changes to functions introduced in the last four commits of the dev branch.

Reply:
Sir, this affects only the slack analysis. I will update the slack driver.

Reply:
@prasadtalasila Infact the driver is correct and doesn't require any change.
https://github.com/prasadtalasila/IRCLogParser/blob/master/slack.py#L143 (Make correct use of the function '''identify_hubs_and_experts'''
But the method in analysis folder doesn't return the correct results. https://github.com/prasadtalasila/IRCLogParser/blob/master/lib/slack/analysis/network.py#L787

Reply:
I am confused. If driver is making calls to wrong function, how did the driver file run at all?

Reply:
I see that we didn't use the values stored in `top_hub, top_keyword_overlap, top_auth` variables. That's probably why we got away without any errors.

Reply:
Yes, i assume that is the reason but even a usage as below
``` 
def func():
    return ret1 
ret1, ret2, ret3 = func() 
```
which seems to the previous case should have raised an error.

Reply:
The python interpreter didn't. :(